### ISSUE TITLE
Pastebin support in bouncer mode

### ISSUE THREAD

**Issue by dasJ** [hifi/heisenbridge#165]
Plumbed rooms create text files in the media storage when too many lines are in one message, similar to how matrix-appservice-irc does it.
The same feature would be desirable for bounced rooms since people on IRC hate it when you paste a lot of lines.

**Comment by poVoq** [comment]
Yeah I also noticed that the plumbed rooms seem to partially drop messages from Matrix with multiple paragraphs.

At least with IRCv3 multi-line support that should not be necessary, but otherwise a paste-bin would be nice indeed,
